Library/Formula/hostname.rb

```diff
@@ -0,0 +1,66 @@
+class Hostname < Formula
+  desc "Utility to set/show the host name or domain name"
+  homepage "https://packages.debian.org/sid/hostname"
+  url "http://ftp.de.debian.org/debian/pool/main/h/hostname/hostname_3.15.tar.gz"
```

Comment:
Please use one of the secure links rather than the plaintext Debian mirrors. Either https://mirrors.kernel.org/debian/pool/main/ or https://mirrors.ocf.berkeley.edu/debian/pool/main/